ui/lib/pki/addon/components/pki-generate-csr.hbs

```diff
@@ -32,9 +32,10 @@
                 <LinkTo @route="keys.key.details" @models={{array @model.backend @model.keyId}}>
                   {{@model.keyId}}
                 </LinkTo>
+              {{else if (value)}}
+                <span>{{"value"}}</span>
```

Comment:
Nice refactor! Just a small change required here, this is just going to pass the string "value" to the `<span>` which I don't think we want to do here. The span also only existed to pass the `tag` class, so this can just be the plain value
```suggestion
                {{value}}
```